modules/administration-guide/partials/ref_devworkspace-specific-metrics.adoc

```diff
@@ -0,0 +1,29 @@
+[id="ref_devworkspace-specific-metrics_{context}"]
+= DevWorkspace-specific metrics
+
+[role="_abstract"]
+This section describes the DevWorkspace-specific metrics exposed by the `devworkspace-controller-metrics` service.
+
+.Metrics
+* *Name:* `devworkspace_started_total` (counter) +
```

Comment:
Questionable format. Is it consistent with the rest of the docs?

Reply:
Sounds great, my latest commit updates the PR to use tables:
![image](https://user-images.githubusercontent.com/83611742/142686895-a41a7a64-83a3-4dc3-981d-05b06e06f160.png)